.github/workflows/prettier-check.yml

```diff
@@ -0,0 +1,19 @@
+name: Prettier Lint
+
+on: [pull_request]
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install Dependencies
+        run: npm install
+        working-directory: ./products/statement-generator
+      - name: Check Prettier Format
+        run: npx prettier --check .
+        working-directory: ./products/statement-generator
```

Comment:
> I'm wondering if it might be easier to read the action if prior to running Install Dependencies, we run `cd products/statement-generator`. then you should be able to run `npm i` and prettier without the working directory lines.

i agree this would be better! just addressed these changes and the step passed